.github/workflows/release.yml

```diff
@@ -1,86 +1,100 @@
 name: Release version
+
 on:
   push:
     tags:
       - 'v*.*.*'
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
i'd also add a manual trigger perhaps, so we do releases on a faster turnaround if needed